### DISCUSSION TITLE
Luna Paint default

### DISCUSSION THREAD

**Discussion by EHMilon** [lunapaint/vscode-luna-paint#198] [Q&A]
I set Luna Paint as the default image viewer. Now how to make VSCode image preview default? Please explain me step by steps?

**Comment by Tyriar** [comment]
https://github.com/lunapaint/vscode-luna-paint/wiki/Customization#dont-open-in-edit-mode-by-default should answer your question

**Reply by EHMilon** [reply]
Tip: If you have already set Luna Paint as the default you can clear this by removing _luna.editor_ from the _workbench.editorAssociations_ setting.
I don't understand this part.
